gradle/libs.versions.toml

```diff
@@ -12,6 +12,7 @@ kotlin-datetime = "0.6.2"
 kotlin-serialization = "1.9.0"
 
 ktor = "3.3.1"
+jetty = "12.0.17"
```

Comment:
when running `gradlew :web:status-history:run` locally it outputs:
```
2025-10-26T01:17:04,262 INFO  [main] org.eclipse.jetty.server.Server(Server.java:581) jetty-12.1.2; built: 2025-10-07T23:18:38.613Z; git: 6cc265bcd6d16f69b8b7f70a7143c8579fe58868; jvm 2
1.0.7+8-LTS-245
```
so there's no need to downgrade that version.